test/Markdown.MAML.Test/Renderer/TextRendererTests.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Markdown.MAML.Model.Markdown;
+using Markdown.MAML.Parser;
+using Markdown.MAML.Renderer;
+
+namespace Markdown.MAML.Test.Renderer
+{
+    public class TextRendererTests
+    {
+        [Fact]
+        public void RendererCreatesAboutTopicString()
```

Comment:
You probably want to test your words wrapping algorithm here as well.
Try to add a long long sentence for this test.